client/src/components/ProfileImg.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+const ProfileImgWrapper = styled.div``;
+
+const ProfileImgSection = styled.div``;
+
+const ProfileImg = () => {
+  const [imageUrl, setImageUrl] = useState<string>("");
+
+  // TODO 이미지가 잘 업로드 되는지 확인하는 함수
+  const checkImg = (e: React.ChangeEvent<HTMLInputElement>): any => {
+    const selectedFile: File | undefined = e.target.files?.[0];
+
+    if (selectedFile) {
+      console.log(selectedFile); // ! 구현 완료 시 삭제되어야 할 코드
+
+    // 파일의 유효성 검사
+      const allowedTypes: string[] = ["image/png", "image/jpeg", "image/jpg"]; // 파일의 타입은 png, jpeg, jpg만 허용
+      if (!allowedTypes.includes(selectedFile.type)) {
+        alert("프로필 이미지는 png, jpeg, jpg 파일만 업로드 가능합니다.");
+        return;
+      }
+      const maxSize: number = 3 * 1024 * 1024; // 파일의 사이즈는 3MB를 넘을 수 없음
```

Comment:
해당 숫자는 상수이므로 컴포넌트 밖으로 빼고 대문자로 쓰시면 좋습니다. 또한, 자동으로 추론되는 타입은 직접 명시하실 필요가 없습니다.
```suggestion
      export const MAX_IMAGE_SIZE = 3 * 1024 * 1024;
```

Reply:
감사합니다! 해당 내용 수정하겠습니다!